style.css

```diff
@@ -1,44 +1,80 @@
 .container {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 0;
+  background: #fff;
+  background-image: url("images/background-image.png");
   display: flex;
-  max-width: 500px;
-  min-height: 100vh;
   flex-direction: column;
-  background-color: white;
+  padding-left: 20px;
 }
```

Comment:
Your design is not 100% in accordance with the Figma template. Please consider reviewing it. Your fonts and margins especially are not matching the design. I am attaching images of both so you can consider them.

![image](https://user-images.githubusercontent.com/94223367/213165417-02f28e2e-8d36-44df-b17f-fbd2dc808bd8.png)

![image](https://user-images.githubusercontent.com/94223367/213165746-3a327cb4-0086-489d-979d-bad166568f17.png)